src/drive/mobile/modules/mediaBackup/duck/tmp.js

```diff
@@ -0,0 +1,62 @@
+/* global cozy */
+/* 
+This file is TEMPORARY. It covers a stack bug and we need to synchronize again some files
+
+!!TO BE DELETED WHEN datas are OK
+
+*/
+import { updateLibraryItem } from 'drive/mobile/lib/media'
+import { CURRENT_UPLOAD_PROGRESS } from './reducer'
+import throttle from 'lodash/throttle'
+import localforage from 'localforage'
+
+export const checkCorruptedFiles = async (
+  photosOnDevice,
+  alreadyUploaded,
+  client,
+  dispatch
+) => {
+  const onProgressUpdate = progress => {
+    // console.log(percent + '%');
+    dispatch({ type: CURRENT_UPLOAD_PROGRESS, progress })
+  }
+  const onThumbnailGenerated = thumbnailUrl => {}
+  let promises = []
+  const seen = (await localforage.getItem('CORRUPTED_FILES')) || false
+  if (seen === false) {
+    const requestFiles = await cozy.client.fetchJSON('GET', '/files/fsck')
+    console.log({ requestFiles })
+    await localforage.setItem('CURREPTED_FILES', requestFiles)
+  }
+  const files = await localforage.getItem('CORRUPTED_FILES')
+  console.log({ files })
+  const ids = files.reduce((memo, value) => {
+    if (value.file_doc && value.file_doc.name) {
+      photosOnDevice.some(photo => {
+        if (photo.fileName === value.file_doc.name) {
+          promises.push(
+            updateLibraryItem(
+              photo,
+              value.file_doc,
+              throttle(onProgressUpdate, 500),
+              onThumbnailGenerated
+            )
+          )
+          return memo.push(photo.id)
+        }
+      })
+    }
+    return memo
+  }, [])
+  try {
+    await Promise.all(promises)
```

Comment:
@y-lohse I make uploading parallel. Do you think it can be an issue ?

Reply:
Mmmh no I think it should be ok!